doc/intl/FR_bzh/README.md

```diff
@@ -0,0 +1,92 @@
+
+### Ar porzhioù
+
+Dogecoin Core a implij dre ziouer ar porzh `22556` evit ar c'hehentiñ par-ouzh-par a zo ezhomm evit sinkronelaat ar Blockchain ouzh ar ".  Ouzhpenn-se e c'haller digeriñ ur porzh JSONRPC. Dre ziouer e vez implijet ar porzh `22555` evit skoulmoù ar rouedad pennañ. Erbediñ a reomp groñs chom hep diskouez ar porzhioù RPC war ar Genrouedad publik. 
+
+| Function | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+## Diorren dibaouez - Moon plan 🌒
+
+Dogecoin Core zo ur meziant frank ha kumuniezhel. Digor eo an argerzh diorren hag a c'haller gwelet evit an holl ; an holl a c'hall gwelet, kendivizout ha labourat war ar meziant.
+ 
+Danvezioù diorren pennañ :
+
+* [GitHub Projects](https://github.com/dogecoin/dogecoin/projects)  a vez implijet evit
+heuliañ al labourioù rakwelet ha war ar stern evit ar stummoù da zont.
+
+* [GitHub Discussion](https://github.com/dogecoin/dogecoin/discussions) a vez implijet evit
+kaozeal eus perzhioù, steuñvet ha n'int ket steuñvet, liammet war un dro ouzh diorroadur ar meziant Dogecoin Core, ar protokolioù dindan hag an oberiant DOGE.
+  
+* [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
+
+### Strategiezh ar stumm
+An niverennoù stumm zo da-heul ar steredeg `` major.minor.patch``.
+ 
+### Skourroù
+Tri seurt skourroù zo er raktres-mañ :
+
+- **master:** Stabil, zo enni ar stumm diwezhañ eus ar *major.minor* handelv.
+- **maintenance:** Stabil, zo enni ar stumm nevesañ eus ar stummoù kent hag a vez dalc'het en oberiant bepred. Furmad : ```<version>-maint``
+- **development:** Stabil, zo enni kod nevez evit ar stummoù rakwelet. Furmad : ```<version>-dev```
+
+*Ar skourroù master ha trezalc'h a c'haller cheñch hepken dre stumm. Ar stummoù steuñvet o do bepred ur skourr diorren hag ar goulennoù pull a rank bezañ kinniget er skourr-mañ.  Ar skourroù trezalc'h zo aze **evit reizhañ ar bugoù** hepken, kinnigit an arc'hwelioù nevez er skourroù diorren gant ar stumm nevesañ.*
+
+## Skodenn 🤝
+
+Ma kavit ur bug pe ma kavit kudennoù gant ar meziant-mañ, kemennit anezhañ en ur implijout [sistem merañ ar c'hudennoù](https://github.com/dogecoin/dogecoin/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D+).
+ 
+Sellit ouzh [heñcher skodennañ](CONTRIBUTING.md) evit gwelet penaos e c'hallit
+Kemer perzh e diorroadur Dogecoin Core. Alies e vez
+[sujedoù o c'houlenn skoazell](https://github.com/dogecoin/dogecoin/labels/help%20wanted)
+lec’h ma vo un efed bras gant ho kemennadennoù ha ma vo priziet-kaer. Wow.
+
+## Kumuniezhoù 🚀🍾
```

Comment:
This entire section (community) should be removed as it no longer exists in recent versions of READMEs and has been corrected for security problems (dead or stolen links).